.github/workflows/tests-dependabot.yml

```diff
@@ -0,0 +1,69 @@
+name: CI (Dependabot branches)
+on:
+  pull_request_target:
+    branches: [master]
+
+env:
+  MIX_ENV: test
+  V3_URL: ${{ secrets.V3_URL }}
+  V3_API_KEY: ${{ secrets.V3_API_KEY }}
+  RECAPTCHA_PUBLIC_KEY: 6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI
+  RECAPTCHA_PRIVATE_KEY: 6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe
+  OPEN_TRIP_PLANNER_URL: ${{ secrets.OPEN_TRIP_PLANNER_URL }}
+
+# Run unit and integration tests, no skipping, minimal caching.
+jobs:
+  ci:
+    name: Run entire CI suite
+    if: ${{ contains(github.head_ref, 'dependabot') }}
```

Comment:
Is there ever the case that a branch could NOT have a head_ref that contains 'dependabot' but DOES have `github.actor == 'dependabot[bot]'`?  The ifs in `tests.yml` reference github.actor, but not the ifs here in `tests-dependabot.yml`.

Reply:
Yeah! I think when I wrote this I wasn't sure of the answer to that question and wanted to be extra safe just in case! 
But for `tests-dependabot.yml` I want the tests to run even if I'm the `github.actor` -- hence it's not included in the ifs here.

Now that I've experimented a bit more on my fork, I probably _could_ leave the `github.actor == 'dependabot[bot]'` check off the `tests.yml` jobs -- it looks like the answer to your question is "probably not"!

Reply:
@hannahpurcell I decided to leave out the github.actor part, but I also edited the contains() to be more specific. otherwise it'll work on _any_ PR branch with dependabot in the name (include this branch!).